docs/website/docs/general-usage/source.md

```diff
@@ -187,6 +187,12 @@ Several data sources are prone to contain semi-structured documents with very de
 MongoDB databases. Our practical experience is that setting the `max_nesting_level` to 2 or 3
 produces the clearest and human-readable schemas.
 
+:::tip TIP
+The `max_table_nesting` parameter at the source level doesn't automatically apply to individual 
+resources when accessed directly (e.g., using `source.resources["resource_1"])`. To make sure it 
+works, either use `source.with_resources("resource_1")` or set the parameter directly on the resource.
```

Comment:
please, add an example how parameter could be set directly in the resource

Reply:
done